src/open_inwoner/contrib/kvk/models.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional, Union
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from simple_certmanager.models import Certificate
+from solo.models import SingletonModel
+
+
+class KvKConfig(SingletonModel):
```

Comment:
Could you add this model to the `django-admin-index.json` fixture? That way it shows up in the admin